.github/workflows/sca.yml

```diff
@@ -79,6 +50,11 @@ jobs:
           retry_wait_seconds: 30
           command: ./dev-tools/install.sh
 
+      - name: Run AutoReview
+        env:
+          FAST_LINT_TEST_CASES: 1
```

Comment:
Does linter make any difference for auto-review tests? These rather don't modify code.

Reply:
👍🏻 
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7407/files#r1382069713